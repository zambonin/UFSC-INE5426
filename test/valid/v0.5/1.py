exec(open('../../../src/scope_manager.py', 'r').read())

j = 0
s_context()
while (j < 10):
    j = (j + 2)
r_context()
s_context()
i = 0
while (i < 10):
    
    temp = (j + i)
    j = temp
    i = (i + 1)
r_context()
j = (j + 0)
