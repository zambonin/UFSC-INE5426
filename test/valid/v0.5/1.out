int var: i, j = 0
for: , < j 10, = j + j 2
do:
for: = i 0, < i 10, = i + i 1
do:
  int var: temp
  = temp + j i
  = j temp
= j + j 0
